operator/v1/types_console.go

```diff
@@ -132,6 +132,9 @@ type ConsoleCustomization struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	AddPage AddPage `json:"addPage,omitempty"`
+	// perspectives allows enabling/disabling of perspective(s) that user can see in the Perspective switcher dropdown.
+	// +optional
+	Perspectives []Perspectives `json:"perspectives"`
```

Comment:
Am i right in thinking that the ID in this list can be unique?

We should add a list type to this for server side apply

Xref https://kubernetes.io/docs/reference/using-api/server-side-apply/#merge-strategy

I suspect this is an atomic list but let's double check

Reply:
We discussed this, ID must be unique within the list
```
// +listType=map
// +listMapKey=id
```